native/packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNav-test.common.js

```diff
@@ -18,11 +18,16 @@
 
 import React from 'react';
 import renderer from 'react-test-renderer';
+
 import BpkHorizontalNav from './BpkHorizontalNav';
+import BpkHorizontalNavItem from './BpkHorizontalNavItem';
+
+jest.mock('./BpkHorizontalNavItem', () => 'BpkHorizontalNavItem');
```

Comment:
Not sure I understand this, you're mocking `BpkHorizontalNavItem` with itself ?!

Reply:
I think it's essentially returning a blank component.

Reply:
I don't get we need it, you don't want and actual `BpkHorizontalNavItem` in the test?

Reply:
(Discussed in person)

Reply:
The outcome of the discussion was: we don't want the full `BpkHorizontalNavItem` there because it's testing the `BpkHorizontalNav` component